tests/console/snapper_jeos_cli.pm

```diff
@@ -15,50 +15,37 @@ use testapi;
 use utils;
 use strict;
 use warnings;
-use power_action_utils 'power_action';
-use version_utils 'is_sle';
+use power_action_utils qw(power_action);
+use version_utils qw(is_sle);
+
+sub rollback_and_reboot {
+
+    my ($self, $rollback_id) = @_;
+    assert_script_run("snapper rollback $rollback_id");
+    assert_script_run("snapper list");
+    power_action('reboot');
+    $self->wait_boot;
+    select_console('root-console');
+    assert_script_run("snapper list");
+}
 
 sub run {
     my ($self) = @_;
 
     select_console('root-console');
     my $file       = '/etc/openQA_snapper_test';
-    my $openqainit = 'openqainit';
-    assert_script_run("snapper create -d $openqainit");
+    my $openqainit = script_output("snapper create -p -d openqainit");
     assert_script_run("touch $file");
-    my $openqalatest = 'openqalatest';
-    assert_script_run("snapper create -d $openqalatest");
-    assert_script_run("snapper list");
-    my $snap_id             = is_sle('15+') ? "'{ print \$1 }'" : "'{ print \$3 }'";
-    my $openqainit_snapshot = script_output("snapper list | grep -w $openqainit | awk $snap_id | tr -d '\\n'");
-    my $latest_snapshot     = script_output("snapper list | grep -w $openqalatest | awk $snap_id | tr -d '\\n'");
-    my $init_snapshot       = script_output("snapper list | grep 'single.*$openqainit' | awk $snap_id | tr -d '\\n'");
-    my $openqarollback      = 'openqarollback';
-    assert_script_run("snapper rollback -d $openqarollback $init_snapshot");
+    my $openqalatest = script_output("snapper create -p -d openqalatest");
     assert_script_run("snapper list");
-    power_action('reboot');
-    $self->wait_boot;
 
-    select_console('root-console');
-    assert_script_run("snapper list");
+    $self->rollback_and_reboot($openqainit);
     assert_script_run("! ls -l $file");
-    assert_script_run("snapper rollback $latest_snapshot");
-    assert_script_run("snapper list");
-    power_action('reboot');
-    $self->wait_boot;
 
-    select_console('root-console');
-    assert_script_run("snapper list");
+    $self->rollback_and_reboot($openqalatest);
     assert_script_run("ls -l $file");
     assert_script_run("rm -v $file");
-    assert_script_run("snapper rollback $openqainit_snapshot");
```

Comment:
To make the logic equal, we need a 
```
$self->rollback_and_reboot($openqainit);
assert_script_run("! ls -l $file");
```

Reply:
this is part which I was trying to explain in a call , I explicitly drop this two lines. Idea behind it was that during `snapper create ` you could create different types of snapshots but according to snapper documentation to get other than "single"  snapshot you need to explicitly specify this. Which we are not doing so this rollback is just repeating same thing which we already checked before

Reply:
Where do we rollback from `test` to `init` with deleted file?

Reply:
I agreed to insert this two lines